### DISCUSSION TITLE
Base Color

### DISCUSSION THREAD

**Discussion by begumbeyza** [google/model-viewer#3257] [Q&A] [answered]
Hi,I have a problem.I try this code https://modelviewer.dev/examples/scenegraph/#changeColor and work it.Thank you.But I want to use different and specific  color.I don't find color code. what are the color codes used here?
Thank you

**Comment by samaneh-kazemi** [comment]
Hi @begumbeyza, thanks for reaching out. Have you tried changing the `data-color` value in that example?

<img width="434" alt="Screen Shot 2022-03-09 at 10 43 46 AM" src="https://user-images.githubusercontent.com/3915613/157510035-b6afb807-2c18-493c-8106-2ffe1e9bd862.png">

**Comment by begumbeyza** [comment]
Thanks for answered.Yes I change but I want to use specific color.For example I want to use hex: AA3000 but I didn't convert for model-viewer.How should I do?

**Comment by samaneh-kazemi** [comment, marked as answer]
We only support RGBA, `setBaseColorFactor(rgba: RGBA) `. More info [here](https://github.com/google/model-viewer/blob/345b9c51b33de1ebca7dbcfa501cbfdf76361635/packages/model-viewer/src/features/scene-graph/pbr-metallic-roughness.ts#L108) 

So you would need to convert your color to RGBA. e.g. in your case try something like `data-color="0.69,0.18,0,1"`

**Comment by begumbeyza** [comment]
I understand now.I ask last quesiton.I research rgba and every color convert don't 0 to 1.How do I do for true convert?
(For example rgba(127, 17, 224, 1)).
Thank you so  much:) and 
I am sorry for disturbing you

**Reply by samaneh-kazemi** [reply]
@begumbeyza just divide the first three numbers by 255. :)

**Reply by begumbeyza** [reply]
Thank you so much:)
